src/components/message/message-card-list/MessagesVirtualizedList.tsx

```diff
@@ -30,14 +30,7 @@ interface Props {
 	computeItemKey?: (idx: number) => React.Key;
 	rowCount: number;
 	itemRenderer: (index: number, message: EventMessage) => React.ReactElement;
-	/*
-		Number objects is used here because in some cases (eg one message / action was 
-		selected several times by different entities)
-		We can't understand that we need to scroll to the selected entity again when
-		we are comparing primitive numbers.
-		Objects and reference comparison is the only way to handle numbers changing in this case.
-	*/
-	scrolledIndex: Number | null;
+	scrolledIndex: number | null;
```

Comment:
It should be a Number object. There are some edge cases when previous scrolledIndex equals to new scrolledIndex, so if it was a number primivite type we couldn't detect it

Reply:
when you choose another event with the same attached message
messages list is reloading
is there any other edge cases?